openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -63,6 +63,31 @@ <h2>{% trans 'Load status' %}</h2>
               </div>
               {% endif %}
               {% endif %}
+              {% if device_data.arp_table %}
+              <h2>{% trans 'ARP table' %}</h2>
+              {% for arp_entry in device_data.arp_table %}
+              <div class="inline-group">
```

Comment:
as mentioned via IM, could you try using a <table> here?

Reply:
Done. I have updated the screenshot